identity/src/lib/config.rs

```diff
@@ -55,3 +56,8 @@ impl Configuration {
         self.jwt_secret.clone()
     }
 }
+
+pub fn configuration() -> Configuration {
```

Comment:
Maybe sth like `get_configuration()` ?

Reply:
I thought about this as well, but then I would also change `rpc_server` to `get_rpc_server`. Otherwise, it does not look consistent to me.